src/common/utils/validateDuplicateFieldValue.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { validateRequired } from '@folio/stripes-acq-components';
+
+export const validateDuplicateFieldValue = async (ky, api, id, value, errorMessage, name, customQuery) => {
+  const errorRequired = validateRequired(value);
+
+  if (errorRequired) {
+    return errorRequired;
+  }
+
+  let query = customQuery || `${name} == "${value}"`;
```

Comment:
it's quite unexpected behavior if `customQuery` is missing. it's also not clear what the `name` stands for. As well as `value`